src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAComServerLibrary.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
With the recent decision to include runtimeconfig.json for these components, do these tests cover those scenarios as well?

Reply:
I haven't yet pushed out those changes.